controllers/tab/workflow/form/FileUploadFormXML.inc.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+/**
+ * @file controllers/tab/workflow/form/FileUploadFormXML.inc.php
```

Comment:
This file and FileCreateFormXML.inc.php are nearly identical... are both necessary?